app/adapters/signal_adapter/outbound/file.rb

```diff
@@ -9,23 +9,17 @@ class File < ApplicationJob
 
       def perform(message:)
         @message = message
-        url = URI.parse("#{Setting.signal_cli_rest_api_endpoint}/v2/send")
-        request = Net::HTTP::Post.new(url.to_s, {
+        uri = URI.parse("#{Setting.signal_cli_rest_api_endpoint}/v2/send")
+        request = Net::HTTP::Post.new(uri, {
```

Comment:
It matters that we pass `URI` not string.

Reply:
Ok, this is just a copy/paste that survived from code you added https://github.com/tactilenews/100eyes/commit/d521f28e919b89b50cc7977eeca941e740cca88c#diff-2e66abeab283cf6d1ae8e78dbcd46410ae0b8eb01361bc44708e1426c6f89f28R31

I don't know why it matters. Maybe, you could further explain